Update TS linting to import root level interfaces

We had some files incorrectly referencing each other which caused errors like this:

```
      "node_modules/@vaadin/vaadin-app-layout/vaadin-app-layout.d.ts(2,15): error TS2307: Cannot find module './@types/interfaces' or its corresponding type declarations.\n",
```

This should never happen again, as we have TS in place. 

*Action point*: update `packages/**/test/typings` to import from the root level entrypoints, not from `src`.

--- a/packages/confirm-dialog/test/typings/confirm-dialog.types.ts
+++ b/packages/confirm-dialog/test/typings/confirm-dialog.types.ts
@@ -1,5 +1,5 @@
 import '../../vaadin-confirm-dialog.js';
-import { ConfirmDialogOpenedChangedEvent } from '../../src/vaadin-confirm-dialog.js';
+import { ConfirmDialogOpenedChangedEvent } from '../../vaadin-confirm-dialog.js';
 
 const assertType = <TExpected>(actual: TExpected) => actual;
 

--- a/packages/split-layout/test/typings/split-layout.types.ts
+++ b/packages/split-layout/test/typings/split-layout.types.ts
@@ -1,4 +1,4 @@
-import '../../src/vaadin-split-layout';
+import '../../vaadin-split-layout';
 
 const assert = <T>(value: T) => value;
 

